.travis.yml

```diff
@@ -13,3 +13,12 @@ sudo: false
 install: composer install --no-interaction
 
 script: composer test
+
+addons:
+  code_climate:
+    repo_token: b10dfa69b04844a8ca9696a5092370008dcc4ac5ca723adcb14d815fbeaebbc7
```

Comment:
We need to encrypt this - https://coderwall.com/p/5mtq6q/encrypt-your-code-climate-repo-token-for-public-repositories-on-travis-ci